ignite/distributed/comp_models/native.py

```diff
@@ -245,7 +245,7 @@ def _dist_worker_task_fn(
             node_rank: int,
             master_addr: str,
             master_port: str,
-            **kw: Any,
+            kw: Any,
```

Comment:
Maybe we can keep kw as kwargs and just remove the comma ?

Reply:
you mean rename it? `kw` -> `kwargs`? I only reverted my change here, I falsely made `kw` to `**kw` and broke a few tests 🙈

Reply:
Ah, OK, sorry, I just saw the last modification. Let's keep as it was =>
```python
kw: Any,
```